docs/pipelines.md

```diff
@@ -548,6 +549,29 @@ performed by the Tekton Controller when a PipelineRun is executed.
 See the [complete list of variable substitutions for Pipelines](./variables.md#variables-available-in-a-pipeline)
 and the [list of fields that accept substitutions](./variables.md#fields-that-accept-variable-substitutions).
 
+### Using the `retries` and `retry-count` variable substitutions
+
+Tekton supports variable substitution for the [`retries`](#using-the-runafter-parameter)
```

Comment:
s/using-the-runafter-parameter/using-the-retries-parameter/

sorry, its very similar 😉